taprpc/universerpc/universe.swagger.json

```diff
@@ -791,6 +791,15 @@
             }
           }
         },
+        "parameters": [
+          {
+            "name": "with_amounts_by_id",
+            "description": "TODO(roasbeef): filter by asset ID, etc?.",
```

Comment:
Is this TODO supposed to be here?

Reply:
Nice catch! The comment was there previously but wasn't mapped to a specific field. Updated that.